tests/app/views/handlers/test_view_submitted_response_pdf.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime, timedelta, timezone
+from unittest.mock import Mock
+
+from werkzeug.exceptions import BadRequestKeyError
+
+from app.data_models import QuestionnaireStore
+from app.questionnaire.questionnaire_schema import QuestionnaireSchema
+from app.views.handlers.view_submitted_response import ViewSubmittedResponseExpired
+from app.views.handlers.view_submitted_response_pdf import ViewSubmittedResponsePDF
+from tests.app.app_context_test_case import AppContextTestCase
+
+
+class TestViewSubmittedResponsePDF(AppContextTestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.schema = QuestionnaireSchema({"post_submission": {"view_response": True}})
+        self.language = "en"
+
+    def set_submitted_at(self, submitted_at):
+        self.storage = Mock()
+        self.storage.get_user_data = Mock(return_value=("{}", 1, submitted_at))
+
+    def test_pdf_not_downloadable(self):
+        submitted_at = datetime.now(timezone.utc) - timedelta(minutes=60)
```

Comment:
Only splitting hairs but would you not test a boundary condition like -46 mins?

Reply:
Fair enough, updated.